src/microsoft-extensions-ai/abstraction-implementations/AbstractionImplementationExamples/LoggingChatClient.cs

```diff
@@ -0,0 +1,36 @@
+using System.Runtime.CompilerServices;
+using Microsoft.Extensions.AI;
+using Microsoft.Extensions.Logging;
+
+public class LoggingChatClient : DelegatingChatClient
```

Comment:
We should probably come up with a different example, given that M.E.AI has the built-in `LoggingChatClient` and associated `UseLogging`.

Reply:
Tracking in this issue - https://github.com/dotnet/ai-samples/issues/79